blazor/rating/templates.md

```diff
@@ -0,0 +1,260 @@
+
+<SfRating EnableSingleSelection=true EnableAnimation=false>
+    <EmptyTemplate>
+        @{
+            @if (context.Index == 0)
+                {
+                <span class='angry emoji'>😡</span>
+                }
+            else if (context.Index == 1)
+                {
+                <span class='disagree emoji'>🙁</span>
+                }
+            else if (context.Index == 2)
+                {
+                <span class='neutral emoji'>😐</span>
+                }
+            else if (context.Index == 3)
+                {
+                <span class='agree emoji'>🙂</span>
+                }
+            else
+                {
+                <span class='happy emoji'>😀</span>
+                }
+        }
+    </EmptyTemplate>
+</SfRating>
+
+<style>
+    .e-rating-item-container:not(.e-rating-selected) .emoji {
+        filter: grayscale(1);
+    }
+</style>
+
+```
+
+![Blazor Rating Component with Emoji Icon](./images/blazor-rating-emoji-icon.png)
+
+## SVG Icon
+
+You can also use SVG icon as a rating item.
+
+Below example demonstrates the SVG Icon of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating EnableAnimation=false>
+    <EmptyTemplate>
+        <svg width="35" height="25" class="e-rating-svg-icon">
+            <rect width="35" height="25" fill="transparent" style="stroke-width:2;stroke:rgb(173,181,189)" />
+        </svg>
+    </EmptyTemplate>
+    <FullTemplate>
+        <svg width="35" height="25" class="e-rating-svg-icon">
+            <defs>
+                <linearGradient id=@("grad"+context.Index.ToString()) x1="0%" y1="0%" x2="100%" y2="0%">
+                    <stop class="start" offset="0%" />
+                    <stop class="end" offset="100%" />
+                </linearGradient>
+            </defs>
+            <rect width="35" height="25" fill=@("url(#grad"+context.Index.ToString()+")") style="stroke-width:2;stroke:rgb(173,181,189)" />
+        </svg>
+    </FullTemplate>
+</SfRating>
+
+<style>
+    .e-rating-svg-icon #grad0 .start {
+        stop-color: #FF0000;
+    }
+
+    .e-rating-svg-icon #grad0 .end,
+    .e-rating-svg-icon #grad1 .start {
+        stop-color: #ff5101;
+    }
+
+    .e-rating-svg-icon #grad1 .end,
+    .e-rating-svg-icon #grad2 .start {
+        stop-color: #ffc801;
+    }
+
+    .e-rating-svg-icon #grad2 .end,
+    .e-rating-svg-icon #grad3 .start {
+        stop-color: #dbe300;
+    }
+
+    .e-rating-svg-icon #grad3 .end,
+    .e-rating-svg-icon #grad4 .start {
+        stop-color: #8bc301;
+    }
+
+    .e-rating-svg-icon #grad4 .end {
+        stop-color: #4eaa01;
+    }
+</style>
+
+```
+
+![Blazor Rating Component with Emoji Icon](./images/blazor-rating-svg-icon.png)
+
+## PNG Image
+
+You can also use PNG image as a rating item.
+
+Below example demonstrates the PNG image of Rating.
+
+```cshtml
+
+@using Syncfusion.Blazor.Inputs
+
+<SfRating>
+    <FullTemplate>        
+        <img src="images/FillStar.png" width="25" height="25" />
```

Comment:
If possible use as base64 image

Reply:
URL for PNG is too large so I just assigned dummy variable and add a comment for it.